src/main/scala/it/pagopa/interop/partyprocess/model/PartyRoleAPIConverter.scala

```diff
@@ -0,0 +1,15 @@
+package it.pagopa.interop.partyprocess.model
+
+import it.pagopa.interop.partyprocess.model.PartyRole.{DELEGATE, MANAGER, OPERATOR, SUB_DELEGATE}
+
+object PartyRoleAPIConverter {
+
+  def fromClientValue(value: String): PartyRole =
+    value match {
+      case "MANAGER"      => MANAGER
+      case "DELEGATE"     => DELEGATE
+      case "SUB_DELEGATE" => SUB_DELEGATE
+      case _              => OPERATOR
```

Comment:
⚠️ a better practice would be to map the precise values and return an error in case of an unexpected result (returning for example an `Either[Throwable, PartyRole]`).
This would prevent future changes to be mapped to OPERATOR.

Reply:
this object was removed some commit ago, because duplicated code from object Conversions->roleToApi